src/chainparams.cpp

```diff
@@ -174,22 +174,17 @@ class CMainParams : public CChainParams
         consensus.nTime_EnforceNewSporkKey = 1566860400;    //!> August 26, 2019 11:00:00 PM GMT
         consensus.nTime_RejectOldSporkKey = 1569538800;     //!> September 26, 2019 11:00:00 PM GMT
 
-        // majorities for block version upgrades
-        consensus.nEnforceBlockUpgradeMajority = 8100; // 75%
-        consensus.nRejectBlockOutdatedMajority = 10260; // 95%
-        consensus.nToCheckBlockUpgradeMajority = 10800; // Approximate expected amount of blocks in 7 days (1440*7.5)
-
         // height-based activations
         consensus.height_last_PoW = 259200;
         consensus.height_last_ZC_AccumCheckpoint = 1686240;
         consensus.height_last_ZC_WrappedSerials = 1686229;
-        consensus.height_start_BIP65 = 1808634;             // 82629b7a9978f5c7ea3f70a12db92633a7d2e436711500db28b97efd48b1e527
+        consensus.height_start_BIP65 = 1808634;             // Block v5: 82629b7a9978f5c7ea3f70a12db92633a7d2e436711500db28b97efd48b1e527
         consensus.height_start_InvalidUTXOsCheck = 902850;
-        consensus.height_start_MessSignaturesV2 = 2153200;  // TimeProtocolV2, Blocks V7 and newMessageSignatures
+        consensus.height_start_MessSignaturesV2 = 2153200;  // height_start_TimeProtoV2
         consensus.height_start_StakeModifierNewSelection = 615800;
-        consensus.height_start_StakeModifierV2 = 1967000;
-        consensus.height_start_TimeProtoV2 = 2153200;       // TimeProtocolV2, Blocks V7 and newMessageSignatures
-        consensus.height_start_ZC = 863787;
+        consensus.height_start_StakeModifierV2 = 1967000;   // Block v6: 0ef2556e40f3b9f6e02ce611b832e0bbfe7734a8ea751c7b555310ee49b61456
+        consensus.height_start_TimeProtoV2 = 2153200;       // Block v7: 14e477e597d24549cac5e59d97d32155e6ec2861c1003b42d0566f9bf39b65d5
+        consensus.height_start_ZC = 863787;                 // Block v4: 5b2482eca24caf2a46bb22e0545db7b7037282733faa3a42ec20542509999a64
```

Comment:
since we're here, could we re-order the lines so that the block versions are grouped and in order (improve readability)?

also, so we need to keep `height_start_MessSignaturesV2` since it has the same value as `height_start_TimeProtoV2`?

Reply:
Alphabetical order is used here.
Not sure that ordering by block version would make it more readable (not all height-based activations are about new block versions).

Reply:
> also, so we need to keep height_start_MessSignaturesV2 since it has the same value as height_start_TimeProtoV2

If we want to unify them into a single parameter we should find a meaningful name (maybe `height_start_BlocksV7` ?

Reply:
will defer and ponder this for a future PR